src/components/views/rooms/EventTile.tsx

```diff
@@ -592,7 +592,7 @@ export class UnwrappedEventTile extends React.Component<EventTileProps, IState>
     private verifyEvent(): void {
         this.doVerifyEvent().catch((e) => {
             const event = this.props.mxEvent;
-            logger.error("Error getting encryption info on event", e, event);
+            logger.error("Error getting encryption info on event", event.getRoomId(), event.getId(), e);
```

Comment:
I think this would still be quite had to read. Personally I'd do something like

```suggestion
            logger.error(`Error getting encryption info on event ${event.getId()} in room ${event.getRoomId()}`, e);
```